lib/ex_hls/client/live/reader.ex

```diff
@@ -159,20 +190,8 @@ defmodule ExHLS.Client.Live.Reader do
   end
 
   defp schedule_next_playlist_check(%{playlist_check_scheduled?: false} = state) do
-    last_segment =
-      state.media_playlist.timeline
-      |> Enum.reverse()
-      |> Enum.find(&match?(%Segment{}, &1))
-
-    assumed_duration_s =
-      case last_segment do
-        %Segment{duration: duration} -> duration
-        nil -> state.media_playlist.target_duration
-      end
-
-    # Schedule the next check in 1/4 of min(target_duration, last_segment.duration)
     playlist_check_timeout_ms =
-      (assumed_duration_s * 250) |> round()
+      (state.media_playlist.info.target_duration * 250) |> round()
```

Comment:
leave a comment why it's 1/4 of TARGET_DURATION instead of 1/2 (as specified in RFC)